### PR TITLE
LPS-114875 Update liferay-npm-scripts to v32.4.0

### PR DESCRIPTION
**Pull request by wincent**
Manual forward from https://github.com/wincent/liferay-portal/pull/312 which had what look to be [spurious Poshi failures](https://github.com/wincent/liferay-portal/pull/312#issuecomment-638422239). ([Reported this](https://liferay.slack.com/archives/CL84ZPHAT/p1591257941277900) in the `#d-quality-assurance` channel so that they can investigate, but not re-running so as to spare resources.)

Original message follows:

---

[Release notes](https://github.com/liferay/liferay-npm-tools/releases/tag/liferay-npm-scripts%2Fv32.4.0).

In short, this brings the [new version of the liferay-npm-bundler-preset-liferay-dev preset](https://github.com/liferay/liferay-npm-tools/releases/tag/liferay-npm-bundler-preset-liferay-dev%2Fv4.4.0), which will enable us to conveniently make use of the new `@clayui/layout` component that was released a couple of weeks ago, without duplication and without having to set up manual bundler imports.

cc @marcoscv-work @eduardoallegrini

### CODE REVIEW

**Comment by wincent** [comment]
ci:reopen

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/2ad0564730...738d3770cd
